tests/model_hub_tests/models_hub_common/test_convert_model.py

```diff
@@ -7,6 +7,10 @@
 from openvino import convert_model
 from openvino.runtime import Core
 
+# set seed to have determenistic generated test data
+# to avoid sporadic issues due to input test data
+np.random.seed(56190)
```

Comment:
This is a legacy function, please use generator
https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html

Reply:
Done